packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/use-flow.md

```diff
@@ -1,26 +1,26 @@
+An app integration using the client-hosted [Sign-In Widget](/code/javascript/okta_sign-in_widget/) with an appropriate [Identity Engine SDK](/docs/guides/oie-embedded-common-download-setup-app/) is a simpler way of implementing embedded authentication where the Widget handles most of the remediation process. You still have control over the step-by-step remediation user experience by configuring authenticators and sign-on policies in your Okta org. With the remedial process managed by the Widget, your app just needs to handle the Interaction Code returned from the completed user interaction. See how this flow is built by reviewing the [Build a use case with the embedded Widget](/docs/guides/oie-embedded-widget-use-case-basic-sign-in) guide.
 
-In addition to configuring the Interaction Code grant type in your [authorization server](#set-up-your-authorization-server) and [your app](#enable-interaction-code-grant-on-an-application), make sure you are using the interaction code flow when you [initialize the Sign-In Widget](/docs/guides/oie-embedded-common-download-setup-app/-/main/#set-up-the-sign-in-widget-and-sdk-for-your-own-app):
+In addition to configuring the Interaction Code grant type in your [authorization server](#set-up-your-authorization-server) and [your app](#enable-interaction-code-grant-on-an-application), make sure you’re using the interaction code flow when you [initialize the Sign-In Widget](/docs/guides/oie-embedded-common-download-setup-app/-/main/#set-up-the-sign-in-widget-and-sdk-for-your-own-app):
 
 * In Okta Sign-In Widget version 7+, Identity Engine is enabled by default, so there are no additional steps to take.
-* If you are using an earlier version than 7, you must explicitly enable Identity Engine features by setting `useInteractionCodeFlow: true` in the config.
+* If you’re using an earlier version than 7, you must explicitly enable Identity Engine features by setting `useInteractionCodeFlow: true` in the config.
 
-> **Note**: Also see [setting up your Okta org](/docs/guides/oie-embedded-common-org-setup/) for typical use case configurations.
+> **Note**: For typical use case configurations, see [setting up your Okta org](/docs/guides/oie-embedded-common-org-setup/).
 
-### Embedded authentication app using a native Identity Engine SDK
+### Apps using an embedded Identity Engine SDK only
```

Comment:
Isn't this supposed to be "embedded authentication app"? "Embedded SDK" doesn't describe the deployment model.

Reply:
@vanngo-okta I don't think so. Redirect and embedded are deployment models as you say, but redirect, embedded widget and embedded sdk are auth types which make more sense here.   https://developer.okta.com/docs/guides/sign-in-overview/main/#choose-your-auth